flink-connectors/flink-connector-hive/pom.xml

```diff
@@ -1024,9 +1024,25 @@ under the License.
 	<profiles>
 		<!-- Activate these profiles with -Phive-x.x.x to build and test against different Hive versions -->
 		<profile>
-			<id>hive-3.1.1</id>
+			<id>hive-3.1.2</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-surefire-plugin</artifactId>
+						<configuration>
+							<!-- disable running test in parallel for hive 3.1.2, otherwise, there'll be some concurrency issues -->
+							<forkCount>1</forkCount>
+							<reuseForks>false</reuseForks>
+							<systemPropertyVariables>
+								<derby.stream.error.file>${project.build.directory}/derby.log</derby.stream.error.file>
+							</systemPropertyVariables>
+						</configuration>
+					</plugin>
+				</plugins>
+			</build>
 			<properties>
-				<hive.version>3.1.1</hive.version>
+				<hive.version>3.1.2</hive.version>
```

Comment:
Why are we now also bumping hive? That should be a separate change.

Reply:
Yes, I agree. That should be a separate change.